speed-transition.lua

```diff
@@ -199,8 +275,10 @@ end
 mp.add_key_binding("ctrl+j", "toggle_speedtrans", toggle)
 mp.add_key_binding("alt+j", "toggle_sub_visibility", toggle_sub_visibility)
 mp.add_key_binding("ctrl+alt+j", "toggle_skipmode", toggle_skipmode)
-mp.add_key_binding("alt++", "increase_speedup", function() change_speedup(0.1) end)
-mp.add_key_binding("alt+-", "decrease_speedup", function() change_speedup(-0.1) end)
+mp.add_key_binding("alt+=", "increase_speedup", function() change_speedup(0.1) end, {repeatable=true})
+mp.add_key_binding("alt+-", "decrease_speedup", function() change_speedup(-0.1) end, {repeatable=true})
```

Comment:
forgot to change = back to + from my setup.
because of the granularity and scope here, this should be repeatable.